infra/gp-cluster-config/templates/14-prometheus-rules.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         - alert: apiserver_podcount
           annotations:
             description: Es muss zumindest 1 Pod da sein
-            summary: Apiserver am {{ .Values.alerts.environment }}-Cluster nicht mehr erreichbar.
+            summary: Apiserver {{`{{ printf "$labels.pod" }}`}} am {{ .Values.alerts.environment }}-Cluster nicht mehr erreichbar.
```

Comment:
Schlussendlich muss {{$labels.pod}} in der resource stehen.
Ich verwende dafür einfach:
{{printf "{{$labels.pod}}"}}

Macht das hier sinn den Pod anzuzeigen? Den gibt es ja nicht?

Außerdem funktioniert in der Summary der Link nicht. Nur in der Description rendert OCP das.